src/providers/krb5/krb5_child.c

```diff
@@ -1395,6 +1556,86 @@ static errno_t k5c_attach_oauth2_info_msg(struct krb5_req *kr,
     return ret;
 }
 
+static errno_t k5c_attach_passkey_msg(struct krb5_req *kr,
+                                      struct sss_passkey_challenge *data)
+{
+    uint8_t *msg;
+    const char *user_verification;
+    int i;
+    size_t msg_len = 0;
+    size_t domain_len = 0;
+    size_t crypto_len = 0;
+    size_t num_creds = 0;
+    size_t cred_len = 0;
+    size_t verification_len = 0;
+    size_t idx = 0;
+    errno_t ret;
+
+    if (data->domain == NULL || data->credential_id_list == NULL ||
```

Comment:
`||` should go in the next line and you should add additional indentation before it.

Reply:
Fixed.